eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -1,6 +1,7 @@
 steps:
   - task: UseDotNet@2
```

Comment:
Did you check the non windows images yet?

Reply:
Not there yet 
https://github.com/Azure/azure-sdk-for-net/pull/17281
https://dev.azure.com/azure-sdk/public/_build/results?buildId=639414&view=logs&j=9cfc3b0b-53b1-524f-85ed-0ef4880bef82&t=277ae693-9b68-5ef5-d666-bb5a558e8cc2